tests/pages/newIdentity.test.tsx

```diff
@@ -0,0 +1,79 @@
+jest.mock("../../src/hooks/useCookies");
```

Comment:
The file naming should follow what you are testing. In this case, it should be `NewIdentity`.

But I think you should rename it too. This is a CRUD heavy application, so adding the actions in the components is crucial imo for scaling. So `AddNewIdentity` would be better I think.

Reply:
OK fixed d25506e